ui/lib/core/addon/components/icon.js

```diff
@@ -43,6 +43,6 @@ export default class Icon extends Component {
   }
 
   get hsIconClass() {
-    return this.size === '24' ? 'hs-icon-xl' : 'hs-icon-l';
+    return this.size === '24' ? 'hs-icon-xlm' : 'hs-icon-l';
```

Comment:
these icons were a rendering a few pixels too large
```js
// icons.scss
.hs-icon-xlm {
  width: 24px;
  height: 24px;
}

.hs-icon-xl {
  width: 28px;
  height: 28px;
}
```

Reply:
Yeah - I think something may have changed? Or at least I noticed in the mount backend form that all of the structure icons were slightly too large. 

There are still some missing from the HDS library so it'd be good to get helios to add whatever doesn't currently exist so we can replace our old icons.